tests/unit_tests/test_data_types.py

```diff
@@ -1148,20 +1185,78 @@ def test_object3d_dict_invalid_string():
 
 @pytest.mark.parametrize(
     "file_name",
-    [
-        "cube.obj",
-        "Box.gltf",
-    ],
+    ["cube.obj", "Box.gltf", "point_cloud.pts.json"],
+)
+def test_object3d_obj_open_file(mock_run, assets_path, file_name):
+    run = mock_run()
+    with open(assets_path(file_name)) as open_file:
+        obj = wandb.Object3D(open_file)
+    obj.bind_to_run(run, "object3D", 0)
+    assert obj.to_json(run)["_type"] == "object3D-file"
+
+
+@pytest.mark.parametrize(
+    "file_name",
+    ["cube.obj", "Box.gltf", "point_cloud.pts.json"],
 )
-def test_object3d_obj(mock_run, assets_path, file_name):
+def test_object3d_from_file_with_path(mock_run, assets_path, file_name):
```

Comment:
one potential alternative to this giant list of tests is removing some of the allowed options for passing in a file. Curious about your opinion

Reply:
It's fine, more tests is (usually) always good ;)